src/auth/interface.js

```diff
@@ -95,6 +95,7 @@ let service;
 export function configure(AuthService, options) {
   PropTypes.checkPropTypes(optionsShape, options, 'property', 'Auth');
   service = new AuthService(options);
+  service.applyMiddleware(options.middleware);
```

Comment:
I _think_ it makes sense to apply middleware in the constructor of the auth service rather than here.  

Or rather, I can't think of a reason to pull it out and not let the auth service handle it, since the implementation understands how to apply middleware and we're passing `options.middleware` into it anyway.  

If middleware were a thing that the service implementations weren't expected to understand, or was a feature of frontend-platform instead of a capability of the service itself, it might belong here.  But in spirit I think we can just let the implementation handle it itself.

For instance, some of our other functions in this file make use of a PubSub system that is extraneous to the core functionality of the service itself, but comes into play if you're using the whole frontend-platform framework together.  This doesn't feel like that - it feels like it belongs to the service implementation, not our framework around it.

Let me know if that doesn't make sense.  It's a bit of a nit, admittedly, but I also feel somewhat strongly. ;)